### DISCUSSION TITLE
Vault Item Labels (tags)

### DISCUSSION THREAD

**Discussion by fone** [bitwarden/clients#7719] [Password Manager]
### Code Contribution Proposal

I started the original feature request for tags in 2018, hoping to get some attention on it here at github.
https://community.bitwarden.com/t/vault-item-labels-tags/132

The community is not looking for workarounds, we've looking for a native tag feature that can use to easily search for items based on those tags. As mentioned in my original post, sysPass had done a fantastic job of implementing this.

**Comment by dwbit** [comment]
Hey @fone let me know if you're proposing a code contribution you would like to write, or just surfacing the original feature request? This space is primarily used for reviewing community code contributions, so happy to pass along any additional info you have.
